### ISSUE TITLE
Investigate sporadic acceptance test failures

### ISSUE THREAD

**Issue by mattherman** [mattherman/MbDotNet#54]
The acceptance tests `CanCreateTcpProxyImposter` passes locally (on Linux), but [fails sporadically during the appveyor build](https://ci.appveyor.com/project/mattherman/mbdotnet/build/1.0.179). It may be due to some issue running the test on Windows or could be something else with the environment the build runs in.

**Comment by mattherman** [comment]
I was able to reproduce the sporadic failures on a Windows machine. I **think** the issue is with the test, but it could mean there is an issue with the TCP proxy feature as well so I'm planning on resolving this before pushing v5.

Mountebank outputs the following error during the test, but the error is present regardless of whether the test has passed or failed:
```
error: [tcp:6001] ::1:65333 transmission error X=> {"code":"EPIPE"}
```

**Comment by mattherman** [comment]
Fixed in #68
